src/app/shared/classes/report-hierarchy-transformer.ts

```diff
@@ -9,9 +10,11 @@ export class ReportHierarchyTransformer {
 
   transform(report: Report): Report {
     const checkpoints: Checkpoint[] = report.checkpoints;
-
     for (const checkpoint of checkpoints) {
-      checkpoint.icon = this.getImage(checkpoint.type, checkpoint.encoding, checkpoint.level);
+      const iconData: IconData = this.getImage(checkpoint.type, checkpoint.encoding, checkpoint.level);
+      checkpoint.icon = iconData.path;
+      checkpoint.iconClass = '';
+      checkpoint.iconClass = iconData.cssClasses;
```

Comment:
Assign iconClass twice?

Reply:
See other comment on the CHECKPOINT_TYPE_STRINGS. I was testing why it was assigning so many -odd's at the end of the cssClass name and forgot to remove it.